src/Utilities/Spherepack.hpp

```diff
@@ -0,0 +1,108 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+/// \file
+/// Provides a C++ interface to the Fortran SPHEREPACK library.
+/**************************************************************************
+This is a modified version of spherepack.  The changes are the following:
```

Comment:
spelling of SPHEREPACK

Reply:
See the last comment in this file before doing anything, might save you some time :)